Add input device stubs (#3411)
ppy/osu-wiki: wiki/Gameplay/Input_device/Graphic_tablet/en.md

> # Graphic Tablet

Reviewer: ```suggestion
# Graphics tablet
```

Reviewer: **the file name should be changed to reflect changes in the title**

> A **graphic tablet** is an input device mainly used for drawing but can be used in osu! as an input device. 

Reviewer: ```suggestion
*See also: [Play Styles](/wiki/Play_Styles)*

A **graphics tablet** (or simply just "tablet") is an input device that is intended for digital drawing and artistry, but can be used as a way to control cursor movement in [osu!standard](/wiki/Game_Modes/osu!). The use of a graphics tablet is a very common play style in osu!standard.
```

> A tablet can be used instead of the mouse to aim. This is a very common play style in osu!

Reviewer: ```suggestion

```

> If you are using a tablet PC or your tablet clicks are not handled correctly, you can enable the "OS TabletPC support" setting in the options menu.

Reviewer: ```suggestion
Notice to tablet players: The use of a Tablet PC or tablet clicks in osu! may sometimes be handled incorrectly To fix this, enable the `OS Tablet PC support` setting in the Options menu.
```
Not sure if my formatting is 100% correct on this, but the main thing is just fixing the POV

> ![OS TabletPC support in osu!](img/1.jpg "OS TabletPC support in osu!") 

Reviewer: this image is not displayed; links to a 404 error

> # Graphics Tablet

Reviewer: ```suggestion
# Graphics tablet
```

*See also: [Play Styles](/wiki/Play_Styles)*

> A **graphics tablet** (or simply just "tablet") is an input device that is intended for digital drawing and artistry, but can be used as a way to control cursor movement in [osu!standard](/wiki/Game_Modes/osu!). 
> The use of a graphics tablet is a very common play style in osu!standard.

Reviewer: ```suggestion
*Note: The use of a graphics tablet is a very common play style in [osu!standard](/wiki/Game_Modes/osu!).*

A **graphics tablet** (or simply just "tablet") is an input device that is intended for digital drawing and artistry, but can be used as a way to control cursor movement in osu!standard. 
```
This was a formatting mistake on my part. Woops.

> Notice to tablet players: The use of a Tablet PC or tablet clicks in osu! may sometimes be handled incorrectly. 
> To fix this, enable the `OS Tablet PC support` setting in the Options menu.

Reviewer: ```suggestion
*Notice to tablet players: The use of a Tablet PC or tablet clicks in osu! may sometimes be handled incorrectly. To fix this, enable the `OS Tablet PC support` setting in the Options menu.*
```
